skbio/stats/power.py

```diff
@@ -1017,10 +1017,10 @@ def _identify_sample_groups(meta, cat, control_cats, order, strict_match):
         m_ids = meta.loc[ids].groupby(cat).groups
         # Checks if samples from the cat groups are represented in those
         # Samples
-        ids_vec = id_vecs = [m_ids[o] for o in order if o in
-                             m_ids]
+        id_vecs = [m_ids[o].values for o in order if o in
```

Comment:
Just tested this PR with a couple of different pandas versions. If we want to avoid bumping our minimum pandas version, we need to drop `.values`. Older versions of pandas return lists here apparently, and index objects are essentially ndarrays as it is, so it works fine without.